src/modules/feedback/components/mapFeedback/MapFeedbackPanel.js

```diff
@@ -0,0 +1,192 @@
+import { html } from 'lit-html';
+import { $injector } from '../../../../injection';
+import { MvuElement } from '../../../MvuElement';
+import css from './mapFeedbackPanel.css';
+
+const Update_Category = 'update_category';
+const Update_Description = 'update_description';
+const Update_EMail = 'update_email';
+const Update_CategoryOptions = 'update_categoryoptions';
+const Update_Geometry_Error_Display = 'update_geometry_error_display';
+const Update_Geometry_Valid = 'update_geometry_valid';
+
+export class MapFeedbackPanel extends MvuElement {
+	constructor() {
+		super({
+			mapFeedback: {
+				state: '',
+				category: '',
+				description: '',
+				email: '',
+				fileId: ''
+			},
+			categoryOptions: [],
+			geometryErrorDisplay: 'none',
+			geometryIsValid: false
+		});
+
+		const {
+			ConfigService: configService,
+			TranslationService: translationService,
+			MapFeedbackService: mapFeedbackService
+		} = $injector.inject('ConfigService', 'TranslationService', 'MapFeedbackService');
+
+		this._configService = configService;
+		this._translationService = translationService;
+		this._mapFeedbackService = mapFeedbackService;
+	}
+
+	onInitialize() {
+		this._getCategorieOptions();
+	}
+
+	async _getCategorieOptions() {
+		try {
+			const categorieOptions = await this._mapFeedbackService.getCategories();
+			this.signal(Update_CategoryOptions, categorieOptions);
+		} catch (e) {
+			console.error(e);
+			this.signal(Update_CategoryOptions, []);
+		}
+	}
+
+	async _saveMapFeedback(mapFeedback) {
+		try {
+			await this._mapFeedbackService.save(mapFeedback);
+		} catch (e) {
+			console.error(e);
```

Comment:
When the MapFeedbackService throws an error we emit also an error notification to inform the user